content/2. Ondersteunende-informatie/4. Techniek/Git/2. Repositories.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 2. Repositories
+draft: false
+---
+> Meer informatie over repositories: [Git Repositories](https://docs.github.com/en/repositories)
+## Wat is een repository?
+Een repository is een locatie waar een project wordt opgeslagen op GitHub. Het fungeert als een centrale opslagplaats voor de broncode, documentatie en andere bestanden die bij een project horen. Via een repository kunnen verschillende gebruikers samenwerken, wijzigingen volgen, en versies beheren. 
+## Hoe maak je een repository aan?
+In de workshop over git wordt uitgelegd hoe een repository aangemaakt kan worden via GitHub, maar een repository kan ook aangemaakt worden via een terminal.
+
+In de terminal navigeer je eerst naar de folder wat jouw root moet worden 
+```
+cd pad/naar/je/project/folder
+```
+
+Daarna voer je het commando uit in die folder om de repository aan te maken
+
+```
+git init
+```
+
+## Public & Private
+Repositories kunnen verschillende instellingen hebben over de zichtbaarheid voor anderen, namelijk `public` en `private`.
+- `public`
+	- Een public repository is voor iedereen zichtbaar en kan ook gecloned worden door iedereen. Public repositories is gebruikelijk voor een open-source project (Een project waar iedereen aan kan bijdragen).
+- `private` 
+	- Een private repository is alleen zichtbaar voor de eigenaar en iedereen die is toegevoegd aan de repository. Dit zijn vaak privé projecten waar informatie instaat die niet publiek toegankelijk moet zijn.
+
+
+## Clonen
+Het clonen van een repository betekent dat je een kopie van de repository naar je lokale machine downloadt. Dit stelt je in staat om lokaal met de code te werken. Wanneer je wijzigingen maakt in de gekloonde versie, kun je deze pushes (zie [[4. Commits en mergen|commits en mergen]]) naar de repository, meestal in een specifieke [[3. Branches |branches]].
+
+In de workshop wordt een uitleg gegeven voor het clonen via GitHub Desktop. Een repository kan ook gekloond worden via de terminal. Dit doe je door de HTTPS URL te kopiëren en in de terminal het volgende commando uit te voeren
+
+> [!warning] Let op
+> Voer dit commando uit in de folder waar je wil dat de repository komt te staan. Dus navigeer eerst naar de folder toe en voer dan het commando uit.
+
+```
+git clone [URL]
+```
+
+Hierin moet `[URL]` vervangen worden met de URL van de repository, gehighlight in de screenshot hieronder. Om dit te openen, moet er gedrukt worden op de groene `<> Code` knop.
+![[clone_repo.jpg]]
+
+## Forken van een repository
+Een repository kan ook 'geforked' worden. Een fork is een kopie van een repository die je kunt gebruiken om wijzigingen aan te brengen zonder de originele code te beïnvloeden. Dit wordt gebruikt voor verschillende zaken
```

Comment:
"is een eigen kopie" ipv "is een kopie". maakt het misschien wat duidelijker dat het iets anders is dan een clone want dat kan je ook zien als een kopie. Oh en er mist nog een punt op het einde